components/HttpsComponent/HttpsRequests.c

```diff
@@ -85,7 +59,7 @@ static void https_get_request(esp_tls_cfg_t cfg, const char *WEB_SERVER_URL, con
         goto exit;
     }
 
-    if (esp_tls_conn_http_new_sync(WEB_SERVER_URL, &cfg, tls) == 1)
+    if (esp_tls_conn_http_new_sync(WEB_SERVER_URL, &cfg, tls) == true)
```

Comment:
I think may be  we need async connection ! , at now we have sync connection  and some time happen bug